cmd/sha3/sha3.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"hash"
+	"io"
+	"os"
+
+	"golang.org/x/crypto/sha3"
+)
+
+// Usage:
+//   sha3 <bytes >hash
```

Comment:
It's moot now because I removed that comment (it's redundant with doc.go), but what was wrong with the spacing? That it was a 2-space indent instead of 4?